lib/puppet/provider/group/groupadd.rb

```diff
@@ -118,10 +127,6 @@ def purge_members
     localmodify('-m', members_to_s(members), @resource.name)
   end
 
-  def member_valid?(user)
-    !!Etc.getpwnam(user)
-  end
```

Comment:
Should we remove `member_valid?` from https://github.com/puppetlabs/puppet/blob/master/lib/puppet/type/group.rb

Reply:
The group call to `member_valid?` is still used by `windows_adsi` provider: https://github.com/puppetlabs/puppet/blob/master/lib/puppet/provider/group/windows_adsi.rb#L60